src/Cli/dotnet/commands/RestoringCommand.cs

```diff
@@ -73,14 +65,19 @@ private static IEnumerable<string> Prepend(string argument, IEnumerable<string>
         private static bool HasArgumentToExcludeFromRestore(IEnumerable<string> arguments)
             => arguments.Any(a => IsExcludedFromRestore(a));
 
-        private static bool IsExcludedFromRestore(string argument) 
-            => argument.StartsWith("-property:TargetFramework=", StringComparison.Ordinal);
+        private static readonly string[] propertyPrefixes = new string[]{ "-", "/" };
+
+        private static bool IsExcludedFromRestore(string argument)
+            => propertyPrefixes.Any(prefix => argument.StartsWith($"{prefix}property:TargetFramework=", StringComparison.Ordinal)) ||
+               propertyPrefixes.Any(prefix => argument.StartsWith($"{prefix}p:TargetFramework=", StringComparison.Ordinal));
 
         //  These arguments don't by themselves require that restore be run in a separate process,
         //  but if there is a separate restore process they shouldn't be passed to it
         private static bool IsExcludedFromSeparateRestore(string argument)
-            => argument.StartsWith("-t:", StringComparison.Ordinal) ||
-               argument.StartsWith("-target:", StringComparison.Ordinal);
+            => propertyPrefixes.Any(prefix => argument.StartsWith($"{prefix}t:", StringComparison.Ordinal)) ||
+               propertyPrefixes.Any(prefix => argument.StartsWith($"{prefix}target:", StringComparison.Ordinal)) ||
+               propertyPrefixes.Any(prefix => argument.StartsWith($"{prefix}consoleloggerparameters:", StringComparison.Ordinal)) ||
+               propertyPrefixes.Any(prefix => argument.StartsWith($"{prefix}clp:", StringComparison.Ordinal));
```

Comment:
Why don't we want to pass console logger parameters to the restore command?

Reply:
When we include it we get output for the separate restore as well as the main target, so it results in having 2 separate "0 errors, build succeeded" messages